app/scripts/modules/appengine/serverGroup/configure/wizard/basicSettings.controller.ts

```diff
@@ -1,17 +1,20 @@
-import { extend, IController, IControllerService, IScope, module } from 'angular';
+import { copy, extend, IController, IControllerService, IScope, module } from 'angular';
 import { StateService } from '@uirouter/angularjs';
 import { set } from 'lodash';
 
-import { NamingService } from '@spinnaker/core';
+import { IArtifact, IExpectedArtifact, NamingService } from '@spinnaker/core';
 
 import { GitCredentialType, IAppengineAccount } from 'appengine/domain/index';
-import { IAppengineServerGroupCommand } from '../serverGroupCommandBuilder.service';
+import { AppengineSourceType, IAppengineServerGroupCommand } from '../serverGroupCommandBuilder.service';
 
 interface IAppengineBasicSettingsScope extends IScope {
   command: IAppengineServerGroupCommand;
 }
 
 class AppengineServerGroupBasicSettingsCtrl implements IController {
+
+  public readonly sourceTypes: string[] = Object.keys(AppengineSourceType).map(k => AppengineSourceType[k as any]);
```

Comment:
Do you need this any more? I'm not seeing it anywhere else.

Reply:
Removed.